foxpuppet/windows/browser/window.py

```diff
@@ -53,5 +89,6 @@ def open_window(self, private=False):
                     *self._file_menu_new_window_button_locator).click()
 
         return self.wait.until(
-            new_browser_window_is_opened(self.selenium, handles_before),
+            expected.new_browser_window_is_opened(
+                self.selenium, handles_before),
```

Comment:
I don't have an issue with this, nor can I see a way around it. Could you perhaps raise this issue with your proposal?